padrino/manifest.yml

```diff
@@ -0,0 +1,3 @@
+---
+applications:
+-  command: bundle exec padrino start -p $PORT
```

Comment:
This manifest looks invalid. According to https://github.com/18F/open-data-maker/blob/cf/manifest.yml, a valid manifest needs a `name` attribute. Does this work?

```yml
---
applications:
- name: cf-hello-word
  command: bundle exec padrino start -p $PORT
```